cli/cmd/errors.go

```diff
@@ -110,3 +111,10 @@ func (e *vulnerabilityPolicyError) validate() bool {
 	e.ExitCode = 0
 	return true
 }
+
+func yikes(msg string) error {
```

Comment:
I LOOOOVE THIS!!

![tenor-156234403](https://user-images.githubusercontent.com/5712253/113424539-8ff80500-938d-11eb-98e7-cf9ee753d9fe.gif)